plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/ElasticsearchQueryRunner.java

```diff
@@ -127,13 +127,10 @@ public static Session createSession()
     public static void main(String[] args)
             throws Exception
     {
-        // To start Elasticsearch:
-        // docker run -p 9200:9200 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.6.2
-
         Logging.initialize();
 
         DistributedQueryRunner queryRunner = createElasticsearchQueryRunner(
-                HostAndPort.fromParts("localhost", 9200),
+                new ElasticsearchServer("docker.elastic.co/elasticsearch/elasticsearch:7.6.2", ImmutableMap.of()).getAddress(),
```

Comment:
Use same version as one used in TestElasticsearch7ConnectorTest (define a constant)